cmd/gitserver/server/serverutil.go

```diff
@@ -107,6 +105,8 @@ var tlsExternal = conf.Cached(func() any {
 	exp := conf.ExperimentalFeatures()
 	c := exp.TlsExternal
 
+	logger := log.Scoped("experimental", "Experimental features").Scoped("tls-external", "Global TLS/SSL settings for Sourcegraph to use when communicating with code hosts.")
```

Comment:
I don't think we need the experimental features scope here:

```suggestion
	logger := log.Scoped("tls-external", "Global TLS/SSL settings for Sourcegraph to use when communicating with code hosts.")
```